_data-prepper/monitoring.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: Monitoring
+nav_order: 2
+---
+
+Metrics in Data Prepper are instrumented using [Micrometer.io](https://micrometer.io/). There are two types of metrics: (1) JVM and system metrics; (2) plugin metrics. Prometheus is used as the default metrics backend.
+
+# JVM and system metrics
+
+JVM and system metrics are Data Prepper instance runtime metrics including metrics on classloaders, memory, garbage collection, threads, etc. See https://micrometer.io/docs/ref/jvm for details.
+
+## Naming
+
+JVM and system metrics in Data Prepper follows pre-defined names in Micrometer.io, e.g. `jvm_classes_loaded`, `jvm_memory_used`.
+
+## Serving
+
+By default, metrics are served from the **/metrics/sys** endpoint on the Data Prepper server. The format is a text Prometheus scrape. This port can be used for any frontend which accepts Prometheus metrics, e.g. [Grafana](https://prometheus.io/docs/visualization/grafana/). The configuration can be updated to serve metrics to other registries like CloudWatch which does not require/host the endpoint but publishes the metrics directly to cloudwatch.
```

Comment:
"... used for any frontend that accepts Prometheus metrics, such as [Grafana] ..."

"The configuration can be updated to serve metrics to other registries (for example, CloudWatch) that do not require or host the endpoint but publish the metrics directly to CloudWatch." I'm confused. Are the metrics always, without exception, published to CloudWatch? Or is CloudWatch the example throughout this statement?

Some other registry does not require or host the endpoint but publishes the metrics directly?

Reply:
This comment is resolved in #2018.